src/sensors/mpu9250sensor.cpp

```diff
@@ -1,6 +1,6 @@
 /*
     SlimeVR Code is placed under the MIT license
-    Copyright (c) 2021 Eiren Rain, S.J. Remington
+    Copyright (c) 2022 Eiren Rain, S.J. Remington, SlimeVR contributors
```

Comment:
```suggestion
    Copyright (c) 2021 Eiren Rain, S.J. Remington, SlimeVR contributors
```

Reply:
actually MPU9250 is working from 30/1/2022, #55 . but who didn't care that. so, I think 2022 is right. however I don't know why fix 2021. can you give me more information?

Reply:
The copyright year is placed either when it's first being made or as a range of when it was made. So it's either 2021 or 2021-2022.